### PR TITLE
Fixes to determining calling ext from call stack

### PR DESCRIPTION
**Pull request by DonJayamanne**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1996,6 +1996,13 @@
             }
         ],
         "viewsContainers": {
+            "activitybar": [
+                {
+                    "id": "jupyter",
```
Comment (DonJayamanne) [inline package.json:2001]: Always available

File: src/client/common/application/extensions.ts [hunk 2]
```diff
@@ -30,32 +30,33 @@ export class Extensions implements IExtensions {
     public async determineExtensionFromCallStack(): Promise<{ extensionId: string; displayName: string }> {
         const stack = new Error().stack;
         if (stack) {
-            const root = EXTENSION_ROOT_DIR.toLowerCase();
-            const frames = stack.split('\n').map((f) => {
-                const result = /\((.*)\)/.exec(f);
-                if (result) {
-                    return result[1].toLowerCase();
```
Comment (DonJayamanne) [inline src/client/common/application/extensions.ts:37]: This would not work on unix

Comment (rchiodo) [reply src/client/common/application/extensions.ts:37]: For some cases. It looks like it won't work with this change either. jupyterExtRoot is still lower case and you're not lowercasing the results.

Comment (rchiodo) [reply src/client/common/application/extensions.ts:37]: Wait did I miss something? It looks like it does check lower case on the filter.

**Review by rchiodo — APPROVED** [review]

**Pull request merged**